lib/components/related_artists/__tests__/layout-test.js

```diff
@@ -0,0 +1,38 @@
+import 'react-native'
+import React from 'react'
+import RelatedArtists from '../'
+
+import renderer from 'react-test-renderer'
+
+jest.mock('../../opaque_image_view.js', () => 'ImageView')
+
+it('lays out correctly', () => {
+  const artists = [ {
+    __id: 0,
+    name: 'Sarah Scott',
+    counts: {
+      for_sale_artworks: 2,
+      artworks: 4,
+    },
+  }]
+
+  // mock nativeEvent with desired dimensions
+  const mockNativeEvent = {
+    nativeEvent: {
+      layout: {
+        width: 768,
+      }
+    }
+  }
+
+  let component = renderer.create(<RelatedArtists artists={artists}/>)
+  let tree = component.toJSON()
```

Comment:
Agreed, the toJSON and tree var are a little confusing. I'd suggest having that helper take the dimensions and do the mocking in there as well.